queries/aws_queries/AWS_Authentication_from_CrowdStrike_Unmanaged_Device_FDREvent.yml

```diff
@@ -0,0 +1,19 @@
+AnalysisType: scheduled_query
+Description: Detects AWS Authentication events with IP Addresses not found in CrowdStrike's AIP List
+Enabled: false
+Query: |
+    SELECT *
+    FROM panther_logs.public.aws_cloudtrail
+    WHERE p_occurs_since('1 days')
+      AND eventName IN ('ConsoleLogin', 'SignIn', 'GetSessionToken')
+      AND eventSource IN ('sts.amazonaws.com', 'signin.amazonaws.com')
+      AND sourceIPAddress NOT IN
+        (
+          SELECT DISTINCT aip
+          FROM crowdstrike_fdrevent
```

Comment:
Doesn't the original sub-query only receive data from `crowdstrike_aidmaster`? If yes, we need to add such a condition.

Reply:
Yup, of course. Pushing in a bit.

Reply:
[I had added it in the rest](https://github.com/panther-labs/panther-analysis/pull/648/files#diff-ea663b430ed68cfe939ad54081e40e347fca526bf46d7637721dc7bda2d68d6bR49), but somehow forgot here.